tests/foreman/cli/test_errata.py

```diff
@@ -163,18 +166,13 @@ def setUp(self):
             'host-collection-id': self.host_collection['id'],
             'organization-id': self.org['id']
         })
-        if bz_bug_is_open('1405434'):
-            # install the custom package for each host
-            for virtual_machine in self.virtual_machines:
-                virtual_machine.run(
-                    'yum install -y {0}'.format(self.CUSTOM_PACKAGE)
-                )
-        else:
-            HostCollection.package_install({
-                'id': self.host_collection['id'],
-                'organization-id': self.org['id'],
-                'packages': [self.CUSTOM_PACKAGE]
-            })
+        for virtual_machine in self.virtual_machines:
+            virtual_machine.run(
+                'yum install -y {0}'.format(self.CUSTOM_PACKAGE)
+            )
+            result = virtual_machine.run(
+                'rpm -q {0}'.format(self.CUSTOM_PACKAGE))
+            self.assertEqual(result.return_code, 0)
```

Comment:
I do not want to rely on installing packages to host collection as I'am testing the errata install, the host collection install packages has full tests for that and many bugs was opened for them, I want if this tests has to fail, only on errata stuff (and not to fail on setUp) that's the idea here.